packages/extension-chakra-storefront/src/setup-app.tsx

```diff
@@ -49,69 +51,22 @@ class ChakraStorefront extends ApplicationExtension<Config> {
 
     extendRoutes(routes: RouteProps[]): RouteProps[] {
         const config = this.getConfig()
-        const extensionRoutes = [
-            {
-                path: '/',
-                component: Pages.Home,
-                exact: true
-            },
-            {
-                path: '/login',
-                component: Pages.Login,
-                exact: true
-            },
-            {
-                path: '/registration',
-                component: Pages.Registration,
-                exact: true
-            },
-            {
-                path: '/reset-password',
-                component: Pages.ResetPassword,
-                exact: true
-            },
-            {
-                path: '/account',
-                component: Pages.Account
-            },
-            {
-                path: '/checkout',
-                component: Pages.Checkout,
-                exact: true
-            },
-            {
-                path: '/checkout/confirmation/:orderNo',
-                component: Pages.CheckoutConfirmation
-            },
-            {
-                path: '/callback',
-                component: Pages.LoginRedirect,
-                exact: true
-            },
-            {
-                path: '/cart',
-                component: Pages.Cart,
-                exact: true
-            },
-            {
-                path: '/product/:productId',
-                component: Pages.ProductDetail
-            },
-            {
-                path: '/search',
-                component: Pages.ProductList
-            },
-            {
-                path: '/category/:categoryId',
-                component: Pages.ProductList
-            },
-            {
-                path: '/account/wishlist',
-                component: Pages.Wishlist
-            }
-        ].filter(({component}) => {
-            return (config.pages || [])[component.displayName] !== false
-        })
+
+        const extensionRoutes = config.routes
```

Comment:
For the chakra-storefront, we no longer hardcodes the routes' paths and now they're configurable.

Reply:
I touched on this a little earlier. Maybe we need to talk about what our pattern is going to be here. I dont know if routes is something that we directly want to be configurable... is it too flexable?